.github/workflows/ci.yml

```diff
@@ -28,7 +28,9 @@ jobs:
         run: rustup show
       - id: cargo-metadata
         run: |
-          WORKSPACE_MEMBERS=$(cargo metadata --format-version=1 --no-deps | jq -c '.packages | map(.name)')
+          # replacing itchysats with taker-electron: dirty fix, as neon and electron require the package name for processing. neon requires the package name to equal the crate name - 
+          # and electron uses the package name as name of the application.
+          WORKSPACE_MEMBERS=$(cargo metadata --format-version=1 --no-deps | jq -c '.packages |.[] | select(.name == "itchysats").name |= "taker-electron" | .name' | jq -s | jq -c '.')
```

Comment:
> Any chance we could handle the new cases in a separate line somehow?
I guess we could remove the itchysats crate from the cargo metadata result and add the taker-electron crate afterwards, but I don't really see the value in that.

The issue here is that the script assumes that the crate name and the folder name are equal. In this case its not - thus without that fix the clippy job would fail.